packages/spaces/src/Spaces.js

```diff
@@ -175,41 +234,26 @@ Spaces.propTypes = {
   spaceIds: PropTypes.arrayOf(PropTypes.string),
   payerIds: PropTypes.arrayOf(PropTypes.string),
   spaces: PropTypes.arrayOf(PropTypes.object),
+  mutliPayerRequired: PropTypes.bool,
 };
 
 Spaces.defaultProps = {
   query: `
-    query($ids: [String!], $payerIDs: [String!], $types: [String!], $page: Int){
-      spaces(ids: $ids, payerIDs: $payerIDs, types: $types, page: $page){
-        totalCount
-        perPage
-        page
-        spaces{
-          id
-          name
-          description
-          link {
-            url
-          }
-          payerIDs
-          parentIDs
-          metadata{
-            name
-            value
-          }
-          images{
-            name
-            value
-          }
-          url
-        }
+    query($ids: [String!], $payerIDs: [String!], $types: [String!], $page: Int, $region: String, $payerSpaceRegion: String, $includeParents: Boolean!){
+      spaces: spaces(ids: $ids, payerIDs: $payerIDs, types: $types, page: $page, region: $region){
+        ... SpaceFragment
+      }
+      payerSpaces: spaces(childIDs: $ids, payerIDs: $payerIDs, types: ["space"], page: $page, region: $payerSpaceRegion) @include(if:$includeParents){
+        ... SpaceFragment    
       }
     }
+    ${SpacesFragment}
   `,
-  variables: { types: ['space'] },
+  variables: { types: ['space'], includeParents: false },
```

Comment:
is there a corresponding change to `slotmachine` that supports this variable?

Reply:
No this is only to use the `@include` directive provided by default for `graphql`